<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uvc_blck</class>
 <widget class="QWidget" name="uvc_blck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1,1">
   <item>
    <widget class="QVideoWidget" name="ui_display" native="true">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>设备名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ui_device"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>分辨率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ui_resolution"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>帧率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ui_frame"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="ui_read">
       <property name="text">
        <string>读取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_write">
       <property name="text">
        <string>写入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_close">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
